lumigator/python/mzai/backend/backend/tests/conftest.py

```diff
@@ -238,11 +307,12 @@ def fake_ray_client():
     return FakeJobSubmissionClient()
 
 
+# FIXME investigate where this is used
```

Comment:
do we still need this comment?

Reply:
Hum. I gave it a quick look, and it seems that the aim was to use it to fake Ray calls. But @peteski22  already uses `requests-mock` for this, so I'll try to remove this fake altogether.

Reply:
Can we remove this comment if not used anymore?